pageserver/src/http/routes.rs

```diff
@@ -87,7 +88,10 @@ fn build_timeline_info(
     let mut info = build_timeline_info_common(timeline)?;
     if include_non_incremental_logical_size {
         info.current_logical_size_non_incremental =
-            Some(timeline.get_current_logical_size_non_incremental(info.last_record_lsn)?);
+            Some(timeline.get_current_logical_size_non_incremental(
+                info.last_record_lsn,
+                CancellationToken::new(),
+            )?);
```

Comment:
Interesting, did not remember this kind of callsite. This is called from async context, could add a TODO about switching these to use spawn_blocking and making the method async?